lib/serverless-tools/deployer/function_deployer.rb

```diff
@@ -62,6 +63,14 @@ def self.docker_deployer(config)
           updater: LambdaUpdater.new(client: Aws::Lambda::Client.new, config: config)
         )
       end
+
+      def self.python_deployer(config)
```

Comment:
The other thing we need is to determine _when_ to use the python builder. The config object will need to return "python" like we do for "ruby" - here: https://github.com/fac/serverless-tools/blob/main/lib/serverless-tools/deployer/function_config.rb#L18